parsers/binplist.py

```diff
@@ -164,7 +164,7 @@ def top_level_object(self):
     return self._ParseObjectByIndex(self.top_level_index, self.object_offsets)
 
   def Open(self, file_obj):
-    data = file_obj.read()
+    data = file_obj.read(-1)
```

Comment:
So it turns out that .read() fails because of some wrapping where Read() expects 2 arguments instead of 1.  It wants a length to read and -1 should be indicating the whole file.  There is probably a better way to handle this, but this was the quick fix to just get it to read the file.